### ISSUE TITLE
Wifi in GameMode is unstable and disconnect

### ISSUE THREAD

**Issue by t1gu1** [ublue-os/bazzite#1963]
### Describe the bug

When I start my PC in game mode, I got frequent Wifi disconnection.
I tested it right aside my router and I even go to my friends house and got the same wifi problem in game mode.

In Desktop mode I don't have the issue.
At least it is way more stable! 
Even if I know that wifi is less performant than when I test it on CachyOS or Windows.

### What did you expect to happen?

I expected at least that the wifi works as the desktop in Game Mode.
The best could be to get better wifi overall.

### Output of `rpm-ostree status`

```shell
State: idle
Deployments:
  ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck-gnome:stable
                   Digest: sha256:8007944db4d5aad6e04952663533e2edd00db394af79216ade2e5f4de7d8fee2
                  Version: 41.20241125 (2024-11-26T03:49:41Z)
                     Diff: 1 added
          LayeredPackages: lua
            LocalPackages: goxlr-utility-1.1.4-1.x86_64

● ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck-gnome:stable
                   Digest: sha256:8007944db4d5aad6e04952663533e2edd00db394af79216ade2e5f4de7d8fee2
                  Version: 41.20241125 (2024-11-26T03:49:41Z)
               LiveCommit: ef3d53fb3b5c76f6d77b5766f0283d007fc14d2188d80f5bd3e2d3000911d066
                 LiveDiff: 1 added
            LocalPackages: goxlr-utility-1.1.4-1.x86_64
                 Unlocked: transient

  ostree-image-signed:docker://ghcr.io/ublue-os/bazzite-deck-gnome:stable
                   Digest: sha256:8007944db4d5aad6e04952663533e2edd00db394af79216ade2e5f4de7d8fee2
                  Version: 41.20241125 (2024-11-26T03:49:41Z)
```

### Hardware

AtomMan G Series

---

More exactly: Atomman G7 PT
![Image](https://github.com/user-attachments/assets/a524e3a7-36cf-468e-8cb2-806e45cb0d58)

### Extra information or context

**The wifi card is**: `MediaTek MT7925 chipset.`

Network controller: MEDIATEK Corp. Device 7925

---

I may suppose that there somethings in the kernel 6.12, since cachyOS seems to handle wifi a bit better?
But for game mode, i got no idea.

I event try the Dev Mode and disabled that setting: (Spoiler: things still not working.)
![Image](https://github.com/user-attachments/assets/c26c93f3-2bdc-4b31-9263-5855b1e02ed6)

---

I Started a Discord thread before opening this issue: https://canary.discord.com/channels/1072614816579063828/1309984773854330971 (Not much info than here, but who knows)

**Comment by atimeofday** [comment]
Hello,

Thank you for the detailed report including the wifi chipset and a variety of things you have tried. I see you also created a thread in the UBlue Discord server for more direct discussion, which seems good to me. 

From other discussions in the server, it looks like kernel 6.12 has been on the horizon for Fedora to be released 'probably within a week' for about three weeks... In theory, Bazzite will be ready to ship it very shortly afterward, so please do keep an eye out for that and let us know whether it solves the issue.

**Comment by t1gu1** [comment]
Thanks for the info!
I'll keep a close eye on the Bazzite [releases](https://github.com/ublue-os/bazzite/releases).

---

By curiosity, I ask ChatGPT if there was change about the `MediaTek MT7925 chipset` in the kernel 6.12.0 to 6.12.4 and it says that:

![Image](https://github.com/user-attachments/assets/ac6b7097-84c0-492d-a388-b5b90ea4699d)

Seems like yes there is. :D
But we may take that as a grain of salt since I don't double check the info.

---

I'll let you what happens when I test the release.
Thank again.

**Comment by t1gu1** [comment]
@atimeofday 

I just test the release in the testing branch: https://github.com/ublue-os/bazzite/releases/tag/testing-41.20241218.1

It use the Kernel `6.12.5-201.bazzite`

From what I test, wifi is now way better!
On desktop it seems to go full speed.
In game mode i don't get disconnected now and it seems stable!

Only bug I can report, is that if I go away in game mode (When it goes to sleep), then it seems to freeze.
From what I test in game mode and desktop mode, it's the only thing i saw.

That being say.
I close the issue because it seems fix now on the testing branch using the `6.12.5-201.bazzite`.
I hope to see the release on stable soon!

Thanks
